wrappers/nixos.nix

```diff
@@ -19,16 +21,21 @@ in {
   options = {
     programs.nixvim = mkOption {
       default = {};
-      type = types.submodule ([
-          {
-            options = {
-              enable = mkEnableOption "nixvim";
-              defaultEditor = mkEnableOption "nixvim as the default editor";
-            };
-            config.wrapRc = mkForce true;
-          }
-        ]
-        ++ shared.topLevelModules);
+      type = types.submoduleWith {
+        shorthandOnlyDefinesConfig = true;
```

Comment:
@roberth I have set this because I saw that [it was set by default when you use the `submodule` type](https://github.com/NixOS/nixpkgs/blob/master/lib/types.nix#L677-L680).
Do you have an opinion on that ?

Reply:
You could probably remove it. By doing so, you enable a bit more syntax to be valid, which might be nice for users. IIRC they can then write `programs.nixvim.imports = [ ... ];`.

> Enabling this only has a benefit when the submodule defines an option named `config` or `options`.

https://nixos.org/manual/nixos/stable/#sec-option-types-submodule

Reply:
Ok, thanks for the answer :)